### PR TITLE
Added github button

### PR DESCRIPTION
**Pull request by madeleine-patience**

### CODE REVIEW

**Comment by Melkeydev** [comment]
@madeleine-patience Can you make this commit against staging instead of GH?
+ add yourself to the contributors file too!

**Review by Melkeydev — CHANGES_REQUESTED** [review]
Few nits

**Review by royanger — CHANGES_REQUESTED** [review]

File: src/components/Nav/BottomButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+function navigateToGithub(){
+window.open("https://github.com/MelkeyOSS/astrostation")
+}
+function BottomButtons(){
+return(
+    <WithTooltip text="Astrostation is an open source project and is open to contributions."> 
+        <div className="fixed bottom-2 ml-32">  
+            <Button  onClick={()=>navigateToGithub()} className="donateButton flex items-center rounded-md bg-violet-700  font-medium text-white shadow-sm focus:outline-none dark:bg-violet-700 dark:text-violet-200" >
+            Github Repo
```
[the remark this answers is not in the document]
Comment (madeleine-patience) [reply src/components/Nav/BottomButtons.tsx]: Text deleted, only icon present.

**Comment by madeleine-patience** [comment]
> @madeleine-patience Can you make this commit against staging instead of GH?
> 
> * add yourself to the contributors file too!

Both of these are done.

**Review by royanger — APPROVED** [review]

**Comment by Melkeydev** [comment]
image
The image seems to be a bit small.
the padding of the button is taking more of the attention than the actual GH logo.
Can we increase the size of the GH icon so it fills in more of the real-estate?

**Comment by madeleine-patience** [comment]
Thank you for the comments, I've updated as you requested, PTAL

**Comment by madeleine-patience** [comment]
I just had a second look this morning- maybe the buttons should be square.

File: src/components/Nav/BottomButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {Button} from "../Common/Button"
+import { AiOutlineGithub } from "react-icons/ai";
+import { WithTooltip } from "../Tooltip"
+import { SiGithub } from "react-icons/si";
+
+function navigateToGithub(){
+window.open("https://github.com/MelkeyOSS/astrostation")
```
Comment (Hacksore) [inline src/components/Nav/BottomButtons.tsx:7]: Hey here is a suggestion, using a `<a>` tag is going to be much better as a semantic HTML element.

So you would need to remove the `navigateToGithub()` function and make the button below use an anchor (`<a>`) tag.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

**Comment by royanger** [comment]
Hey @madeleine-patience, we noticed that the new component that holds the bottom buttons is missing some left padding. The Donate button is butting up against the edge of the screen. Compare that to the production release to see the difference.

**Comment by madeleine-patience** [comment]
@royanger padding re-added and @Hacksore I changed the navigate function to an anchor tag instead.

**Pull request merged**